benches/dalek_benchmarks.rs

```diff
@@ -14,8 +14,17 @@ use curve25519_dalek::constants;
 use curve25519_dalek::scalar::Scalar;
 
 static BATCH_SIZES: [usize; 5] = [1, 2, 4, 8, 16];
-static MULTISCALAR_SIZES: [usize; 13] = [1, 2, 4, 8, 16, 32, 64, 128, 256, 384, 512, 768, 1024];
-
+static CONSTTIME_MULTISCALAR_SIZES: [usize; 13] = [1, 2, 4, 8, 16, 32, 64, 128, 256, 384, 512, 768, 1024];
+static VARTIME_MULTISCALAR_SIZES: [usize; 32] = [
+    1, 2, 4, 8,
+    16, 32, 64, 128,
+    152, 181, 215, 256,
+    304, 362, 431, 512,
+    609, 724, 861, 1024,
+    1218, 1448, 1722, 2048,
+    2435, 2896, 3444, 4096,
+    4871, 5793, 6889, 8192
+];
```

Comment:
Where do these sizes come from? Is there a reason not to use powers-of-two?

Reply:
This is powers-of-two with 3 values in-between each powers to give more fine-grained benchmarks. I used it at one point to help me find the thresholds, but maybe we can simplify it now.